### PR TITLE
Feature/add camera info

### PR DESCRIPTION
**Pull request by wep21**
Add camera info publisher and sample camera info yaml.

### CODE REVIEW

**Comment by wep21** [comment]
@flynneva Could you also check this?

**Comment by flynneva** [comment]
@wep21 these PR's are great! Keep them coming 😃 

LGTM

File: config/params.yaml [hunk 1]
```diff
@@ -8,3 +8,4 @@
       image_width: 640
       image_height: 480
       camera_name: "test_cam"
+      camera_info_url: "package://usb_cam/config/camera_info.yaml"
```
Comment (flynneva) [inline config/params.yaml:11]: did you want to use this param within the code?

**Comment by flynneva** [comment]
@wep21 shoot I just realized you have `test_cam` and `test_camera` frame ids. did you want these to match?

**Comment by wep21** [comment]
@flynneva Oh, I'm sorry... I think it's better to have the same name. I appreciate if you colud match them.

**Pull request merged**